lib/Models/Cesium.js

```diff
@@ -810,20 +814,30 @@ Cesium.prototype.pickFromScreenPosition = function(screenPosition) {
 Cesium.prototype.pickVectorFeatures = function(screenPosition) {
     // Pick vector features
     var vectorFeatures = [];
-    var picked = this.scene.drillPick(screenPosition);
-    for (var i = 0; i < picked.length; ++i) {
-        var id = picked[i].id;
+    var pickedList = this.scene.drillPick(screenPosition, 100);
```

Comment:
ohh good catch. Just using 101 instead should solve it though, right?

Reply:
It won't show the total number count that it used to. Is it no longer possible to get the total number of picked features?

Reply:
Unfortunately I added that ages ago and don't 100% remember why, but I think it was because picking was locking up the browser in some cases because there were a ridiculous number of features being picked. I'll look into it more.